client/task.go

```diff
@@ -385,7 +385,12 @@ func (t *task) Delete(ctx context.Context, opts ...ProcessDeleteOpts) (*ExitStat
 			// https://github.com/containerd/containerd/issues/7357
 			break
 		}
-		fallthrough
+		_, err := t.client.TaskService().Kill(ctx, &tasks.KillRequest{
```

Comment:
I think the invoker can call kill before delete, which helps keep delete simple.

Reply:
ok.thank you. use ctr t kill and ctr t delete will delete it.